source/core/src/main/com/csse3200/game/screens/CombatScreen.java

```diff
@@ -41,10 +41,11 @@
  */
 public class CombatScreen extends ScreenAdapter {
```

Comment:
this class should probably extend pausableScreen, even tho the combat is turn based, it would enable overlays so that you can clean the game while in the combat screen.